### PR TITLE
Final review of Tor section

### PR DESCRIPTION
**Pull request by anselld**
Hi @leet4tari, I've reviewed your Tor section and have suggested a small change to the note about Tor vs services. Please let me know what you think.

### CODE REVIEW

File: src/protocols/tor-or-not-too-tor/tor-or-not-too-tor.md [hunk 1]
```diff
@@ -145,12 +145,16 @@ Date accessed: 2019&#8209;05&#8209;16.
...
 [3]: https://3g2upl4pq6kufc4m.onion/
-"DuckDuckGo Search Engine - link will not work unless Tor Browser is used"
+"DuckDuckGo Search Engine - 
+link will not work unless 
+Tor Browser is used"
 
 [[4]] Tor Project: Check [online]. Available: <https://check.torproject.org/>. **Note:** This link will not work unless the Tor Browser is used. Date accessed: 2019&#8209;05&#8209;16
 
 [4]: https://check.torproject.org/
-"Tor Project: Check - link will not work unless Tor Browser is used"
```
Comment (leet4tari) [inline src/protocols/tor-or-not-too-tor/tor-or-not-too-tor.md]: The Tor checker will work, but report when you not using Tor, so the note below is incorrect. I think we can remove the note. Maybe we can make the description a little better. "Use to report if You using Tor or not.

Comment (anselld) [reply src/protocols/tor-or-not-too-tor/tor-or-not-too-tor.md]: Hi @leet4tari, yes, please remove the note under [[4]] as well as the text after 'Check', under [4].

Where is the description that you are referring to?

Comment (leet4tari) [reply src/protocols/tor-or-not-too-tor/tor-or-not-too-tor.md]: Thinking we go from "Tor Project: Check [online]. Available" to "Tor Project: Check if you using Tor [online]. Available" ... Thou that sounds so ugly?

Comment (anselld) [reply src/protocols/tor-or-not-too-tor/tor-or-not-too-tor.md]: Hi @leet4tari, I see that the note is after both [[3]] and [[4]].
Would "Tor Project: Check if you are using Tor" be correct for [[4]] and [4]?
If so, what should appear in [[3]] and [3]?

Comment (leet4tari) [reply src/protocols/tor-or-not-too-tor/tor-or-not-too-tor.md]: Hey @anselld , [[3]] is an ```.onion``` site, which only works with Tor, where [[4]] is just a check to warn you if you not using Tor or letting you know that you are using Tor. Works on both Tor and none Tor setups.

Comment (anselld) [reply src/protocols/tor-or-not-too-tor/tor-or-not-too-tor.md]: Hi @lee4tari, thanks for the clarification.

**Pull request merged**